packages/iris-grid/src/mousehandlers/IrisGridTokenMouseHandler.ts

```diff
@@ -43,7 +44,10 @@ class IrisGridTokenMouseHandler extends GridMouseHandler {
     }
 
     const renderState = grid.updateRenderState();
-    const tokensInCell = renderer.getTokenBoxesForVisibleCell(
+    const textCellRenderer = renderer.getCellRenderer(
+      'text'
+    ) as TextCellRenderer;
+    const tokensInCell = textCellRenderer.getTokenBoxesForVisibleCell(
```

Comment:
Get cell render type from the model, then get the renderer. Return if it's not a TextCellRenderer.

Ideally we'd put `getTokenBoxesForVisibleCell` into a new interface, and then `TextCellRenderer` would implement that, e.g.:
```
interface TokenBoxesCellRenderer {
  getTokenBoxesForVisibleCell(...): ...;
}

isTokenBoxesCellRenderer(cellRenderer: CellRenderer): cellRenderer is TokenBoxesCellRenderer {
  return (cellRenderer as TokenBoxesCellRenderer)?.getTokenBoxesForVisibleCell !== undefined;
}
```

And then here it would be something like:
```
const cellRendererType = model.rendererForCell(column, row);
const cellRenderer = renderer.getCellRenderer(cellRendererType);
if (!isTokenBoxesCellRendrerer(cellRenderer)) {
  return;
}
...
```